autoload/neomake/makers/ft/markdown.vim

```diff
@@ -19,6 +19,13 @@ function! neomake#makers#ft#markdown#proselint()
                 \ }
 endfunction
 
+function! neomake#makers#ft#markdown#writegood()
+    return {
+                \ 'args': ['--parse'],
+                \ 'errorformat': '%f:%l:%c:%m'
```

Comment:
Thanks for letting me know - is it possible for you to point me to documentation? I'd like to know how %W works. So far I've used other plugins as guidance.